lib/domgen/jpa/model.rb

```diff
@@ -655,13 +660,16 @@ def pre_verify
         end
 
         entity.queries.select { |query| query.jpa? && query.jpa.no_ql? }.each do |query|
+          ignore_deleted_at_str = 'IgnoringDefaultCriteria'
+          query.jpa.ignore_default_criteria = query.name.include?(ignore_deleted_at_str)
```

Comment:
Rather than using include? you should use regex to check that the string should appear at end. Maybe `query.name ~= /^.* IgnoringDefaultCriteria$/`